core/lib/multivm/src/glue/tracers/mod.rs

```diff
@@ -57,6 +58,10 @@ pub trait IntoLatestTracer<S: WriteStorage, H: HistoryMode> {
     fn latest(&self) -> crate::vm_latest::TracerPointer<S, H::Vm1_5_0>;
 }
 
+pub trait IntoEraVmTracer<S: WriteStorage, H: HistoryMode> {
+    fn era_vm(&self) -> Box<dyn crate::era_vm::tracers::traits::VmTracer<S>>;
```

Comment:
I find the method name a bit confusing. Maybe `into_era_vm`?

Reply:
Also, maybe implementing `From` is enough? I'm not 100% sure, just a suggestion.

Reply:
If it's OK with you, I may check the branch out and try it locally.

Reply:
That is the way they do it for the other vms, I think it is better to keep the same type of implementation, that doesn't mean your suggestion won't work though.

Reply:
Yes, if it's already done this way everywhere else it's good to be consistent.